package.json

```diff
@@ -15,33 +14,22 @@
   },
   "scripts": {
     "start": "electron .",
+    "clean": "rm -rf release doc/Sia-UI node_modules app/dependencies app/config.json **/*.swp npm-debug.log",
     "debug": "node-inspector & electron --debug=5858 . & xdg-open http://127.0.0.1:8080/debug?ws=127.0.0.1:8080&port=5858",
-    "clean": "rm -rf build doc/Sia-UI node_modules app/dependencies app/config.json **/*.swp npm-debug.log",
-    "fresh": "npm run clean && npm install && npm start",
-    "lint": "jshint . --verbose --exclude-path=.jshintignore",
     "doc": "jsdoc -c .jsdocrc",
-    "postinstall": "npm run sia-repo && npm run sia-import",
-    "sia-repo": "go get -u github.com/NebulousLabs/Sia/...",
-    "sia-import": "(test -d app/dependencies/Sia || mkdir -p app/dependencies/Sia) && cp $GOPATH/bin/siad app/dependencies/Sia",
-    "sia-kill": "killall siad; while pgrep siad; do sleep .5; done;",
-    "sia-reimport": "npm run sia-repo && npm run sia-kill && npm run sia-import",
-    "build": "npm run build-cmd -- --all",
-    "build-lin64": "npm run build-cmd -- --platform=linux --arch=x64",
-    "build-lin32": "npm run build-cmd -- --platform=linux --arch=ia32",
-    "build-win64": "npm run build-cmd -- --platform=win32 --arch=x64",
-    "build-win32": "npm run build-cmd -- --platform=win32 --arch=ia32",
-    "build-mac": "npm run build-cmd -- --platform=darwin --arch=x64",
-    "build-cmd": "electron-packager . Sia-UI --version=0.30.0 --prune --ignore='doc|build' --out=build --icon=app/assets/logo/sia.svg"
+    "lint": "jshint . --verbose --exclude-path=.jshintignore",
+    "package": "make package",
+    "package-linux64": "make package-linux64",
+    "package-linux32": "make package-linux32",
+    "package-win64": "make package-win64",
+    "package-win32": "make package-win32",
+    "package-osx": "make package-osx"
```

Comment:
Oh, well I guess that works. Why not describe these npm scripts in the Developers.md instead so as to avoid instructions about both `npm run scripts` and `make scripts`?